Makefile

```diff
@@ -525,14 +527,16 @@ $(JS_CONFIG): chaise-config-sample.js
 .make-detailed-asset-block: $(DETAILED_SHARED_CSS_DEPS) $(DETAILED_CSS_DEPS) $(DETAILED_CSS_SOURCE) $(DETAILED_SHARED_JS_DEPS) $(DETAILED_JS_DEPS) $(DETAILED_JS_SOURCE) $(JS_CONFIG)
 	> .make-detailed-asset-block
 	for file in $(DETAILED_SHARED_CSS_DEPS) $(DETAILED_CSS_DEPS); do \
-		echo "<link rel='stylesheet' type='text/css' href='../$$file'>" >> .make-detailed-asset-block ; \
+		checksum=$$($(MD5) $$file | awk '{ print $$1 }') ; \
+		echo "<link rel='stylesheet' type='text/css' href='../$$file?v=$$checksum'>" >> .make-detailed-asset-block ; \
 	done
 	for file in $(DETAILED_CSS_SOURCE); do \
 		checksum=$$($(MD5) $$file | awk '{ print $$1 }') ; \
 		echo "<link rel='stylesheet' type='text/css' href='../$$file?v=$$checksum'></script>" >> .make-detailed-asset-block ; \
```

Comment:
@chiragsanghvi This is unrelated to this issue, but there's a random `</script>` tag for this CSS file; could you remove it?